### PR TITLE
Blockchain accountService implementation

### PR DESCRIPTION
**Pull request by scuevas-bc**

### CODE REVIEW

**Comment by scuevas-bc** [comment]
Hi @timmolter, would you or anyone of your team mind taking a look at this PR ?

**Comment by scuevas-bc** [comment]
Sorry for the insistence but can someone help me with this PR ? @walec51 ? or any other reviewer ?

File: xchange-blockchain/src/test/resources/__files/withdrawHistory-success.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  {
+    "withdrawalId": "string",
+    "amount": 12.23,
+    "fee": 0.0005,
+    "currency": "BTC",
+    "beneficiary": "string",
+    "state": "REJECTED",
+    "timestamp": 1592830770594
+  }
+]
```
Comment (timmolter) [inline xchange-blockchain/src/test/resources/__files/withdrawHistory-success.json:11]: to be consistent you should put these json files in a path where the deleted `address.json` was.

**Comment by timmolter** [comment]
looks really good. Just address my one comment please.

File: xchange-blockchain/src/test/resources/__files/withdrawHistory-success.json (hunk 1 above)
Comment (scuevas-bc) [reply xchange-blockchain/src/test/resources/__files/withdrawHistory-success.json:11]: Hi @timmolter first of all thanks for the review.
Regarding your comment, we put the json files in __files directory as recommended by WireMock documentation (“To read the body content of a file, place the file in the __files directory.” https://wiremock.org/docs/stubbing/).
We also saw that other integrations use the same approach. But anyway if you think we should change it, we can change it.

**Comment by timmolter** [comment]
It looks like the build failed due to this merge. Can you check it out please?

https://github.com/knowm/XChange/runs/6810546690?check_suite_focus=true

It looks like a unit test is reaching out over the internet. Instead, we consider these integration tests and the class names need to end with *Integration.java.

**Pull request merged**